activerecord/test/cases/migration/column_attributes_test.rb

```diff
@@ -9,6 +9,12 @@ class ColumnAttributesTest < ActiveRecord::TestCase
 
       self.use_transactional_tests = false
 
+      def test_add_column_with_an_unsupported_option
+        assert_raises ArgumentError do
+          add_column "test_models", "command", :string, index: true
```

Comment:
It's a bit unclear that `index` isn't allowed:
```suggestion
          add_column "test_models", "command", :string, unknown: true
```